snuba/query/conditions.py

```diff
@@ -115,12 +123,24 @@ def binary_condition(
     return FunctionCall(alias, function_name, (lhs, rhs))
 
 
+binary_operators = [opr for opr in FUNCTION_TO_OPERATOR] + [
```

Comment:
This includes also some unary operator like `IS NULL` and `IS NOT NULL`

Reply:
I know but it expects 2 arguments so it shouldn't match the unary operator expressions.

Reply:
I changed it. Easier that way.